pkgs/top-level/python-packages.nix

```diff
@@ -1178,6 +1178,45 @@ in {
     };
   };
 
+  couchpotato = buildPythonPackage rec {
+    name = "couchpotato-${version}";
+    version = "3.0.1";
+    disabled = isPy3k;
+
+    src = pkgs.fetchurl {
+      url = "https://github.com/CouchPotato/CouchPotatoServer/archive/build/${version}.tar.gz";
+      sha256 = "1xwjis3ijh1rff32mpdsphmsclf0lkpd3phpgxkccrigq1m9r3zh";
+    };
+
+    phases = [ "unpackPhase" "patchPhase" "installPhase" "postInstall" ];
+
+    postPatch = ''
+      substituteInPlace CouchPotato.py --replace "dirname(os.path.abspath(__file__))" "os.path.join(dirname(os.path.abspath(__file__)), '../${python.sitePackages}')"
+    '';
+
+    installPhase = ''
+      mkdir -p $out/bin/
+      mkdir -p $out/${python.sitePackages}/
+
+      cp -r libs/* $out/${python.sitePackages}/
+      cp -r couchpotato $out/${python.sitePackages}/
+
+      cp CouchPotato.py $out/bin/
```

Comment:
having a binary with an extension is rather ugly. Maybe we want to rename it instead to `couchpotato` ?